packages/generator-one-app-module/README.md

```diff
@@ -41,6 +41,11 @@ Doing this, you're able to extend the `generator-one-app-module` generator, enab
 
 [More on composing generators here](https://yeoman.io/authoring/composability.html).
 
+
+### Bundling modules
+
+For more information about what happens after bundling your module, read [Bundling modules](./docs/Bundling-Modules.md)
```

Comment:
i think this should just link to the `one-app-bundler` docs

Reply:
Anything specific to bundling modules that is only relevant when using the generator ?

Reply:
i wanted to provide a link for anyone if they wanted to review what happens when the docs are bundled. I have linked one-app-bundler in the docs

Reply:
thats fair